diesel_cli/src/database.rs

```diff
@@ -34,11 +34,21 @@ impl Backend {
                 Backend::Mysql,
             #[cfg(feature="sqlite")]
             _ => Backend::Sqlite,
-            #[cfg(not(feature="sqlite"))]
+            #[cfg(all(not(feature="sqlite")), all(feature="postgres", feature="mysql"))]
             _ => {
-                panic!("{:?} is not a valid database URL. It should start with \
+                panic!("`{}` is not a valid database URL. It should start with \
                         `postgres://` or `mysql://`", database_url);
+            },
+            #[cfg(all(not(feature="sqlite")), feature="postgres")]
```

Comment:
I wonder if we can use `cfg_if!` here? If not, you'll need this to be `all(feature="postgres", not(feature="mysql"))` to avoid warnings.